app/views/about/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="grid-container">
+    <div class="grid-row grid-gap">
+        <main class="usa-layout-docs__main desktop:grid-col-9 usa-prose usa-layout-docs" id="main-content">
+            <div class="grid-row margin-top-2 margin-right-0">
+                <div class="grid-col-12">
+
+                    <h1>About</h1>
+                    <p>
+                    We created this dashboard to provide a faster way to find and analyze Privacy Act notices in the <a href="https://www.federalregister.gov/">Federal Register</a>.
```

Comment:
I know we have some big questions about "we" but for now can we say "[18F](https://18f.gsa.gov/) with [10x](https://10x.gsa.gov/) funding created this..."